server/src/concepts/offer/controllers/index.js

```diff
@@ -0,0 +1,18 @@
+import OfferValidation from "../model/OfferValidation.js";
+import { getOffers } from "../useCases/getOffers.js";
+
+export const getOffersHere = async (req, res) => {
+  const validationResult = OfferValidation(req.body);
```

Comment:
we do not need validation for getting items from DB, especially that we do not send anything in request body